### PR TITLE
Report offending char in error message.

### PR DESCRIPTION
**Pull request by epost**

### CODE REVIEW

File: src/Text/Parsing/Parser/String.purs [hunk 1]
```diff
@@ -48,7 +48,7 @@ satisfy :: forall m. (Monad m) => (Char -> Boolean) -> ParserT String m Char
 satisfy f = try do
   c <- anyChar
   if f c then return c
-         else fail "Character did not satisfy predicate"
+         else fail $ "Character '" ++ (fromChar c) ++ "' did not satisfy predicate"
```
Comment (paf31) [inline src/Text/Parsing/Parser/String.purs]: Looks good, thanks! Could you please remove the unnecessary parens though?

Comment (epost) [inline src/Text/Parsing/Parser/String.purs]: Oh, the shame! :)

**Comment by beckyconning** [comment]
:+1:

**Comment by paf31** [comment]
Thanks!

**Pull request merged**